sormas-ui/src/main/java/de/symeda/sormas/ui/contact/AbstractContactGrid.java

```diff
@@ -114,14 +114,9 @@ protected void initColumns() {
 		diseaseShortColumn.setSortProperty(ContactIndexDto.DISEASE);
 
 		Column<IndexDto, String> visitsColumn = addColumn(entry -> {
```

Comment:
It still creates a new column instead of base on a DTO attribute (which was requested by the ticket: "Use that new attribute to configure that formatted column").

Technically it is not needed to align that. It's cleaner, but also causes also changes in column properties. @MateStrysewske what do you think.

Reply:
Since two DTO attributes are involved here, how would you technically create a column based on the DTO though? I wouldn't know how to implement that right now (without putting too many thoughts into it).

Reply:
You can decide which attribute should show the formatted information. If the formatted information would be part of the DTO (see Implementation Details discussion) then there would be a dedicated attribute to format.